src/app/loadout/fashion/FashionDrawer.tsx

```diff
@@ -495,7 +493,6 @@ function FashionSocket({
   onRemovePlug,
 }: {
   bucketHash: number;
-  plugHash: number | undefined;
```

Comment:
We have the plug, which we can get the plug hash from, so I figure we don't need this.